get_started/what_is_citus.rst

```diff
@@ -3,34 +3,33 @@
-across multiple nodes in a cluster. Because Citus is an extension (not a fork)
-to Postgres, when you use Citus, you are also using Postgres.  You can leverage
+The Citus database is an open source extension to Postgres that gives you all the greatness of Postgres, at any scale—from a single node to a large distributed database cluster. Because Citus is an extension (not a fork)
+to Postgres, when you use Citus, you are also using Postgres. You can leverage
 the latest Postgres features, tooling, and ecosystem.
 
-Citus transforms Postgres into a distributed database with features like
-sharding, a distributed SQL engine, reference tables, and distributed tables.
+With Citus you get distributed Postgres features like
+sharding, distributed tables, reference tables, a distributed query engine, columnar storage—and as of Citus 11.0, the ability to query from any node.
 The Citus combination of parallelism, keeping more data in memory, and higher
 I/O bandwidth can lead to significant performance improvements for multi-tenant
 SaaS applications, customer-facing real-time analytics dashboards, and time
 series workloads.
 
 **Two Ways to Get Citus:**
 
-1. Open source: You can `download Citus <https://www.citusdata.com/download/>`_
-   open source, or visit the `Citus repo <https://github.com/citusdata/citus>`_
+1. **Open source**: Citus is 100% open source. You can `download Citus <https://www.citusdata.com/download/>`_
+   open source, or to see the source code and build it yourself, visit the `Citus repo <https://github.com/citusdata/citus>`_
    on GitHub.
-2. Managed database service: Citus is available on Azure as `Hyperscale (Citus)
-   <https://docs.microsoft.com/azure/postgresql/hyperscale/>`_, a
-   built-in deployment option in the Azure Database for PostgreSQL managed
-   service.
+2. **Managed service**: The Citus database is available as a managed service in the cloud with `Azure Cosmos DB for PostgreSQL
+   <https://learn.microsoft.com/azure/cosmos-db/postgresql/introduction/>`_, formerly known as Hyperscale (Citus) in Azure Database for PostgreSQL.
 
 .. _how_big:
 
-How Far Can Citus Scale?
-------------------------
+Citus Gives You Postgres At Any Scale
+-------------------------------------
 
-Citus scales horizontally by adding worker nodes, and vertically by upgrading workers/coordinator. In practice our customers have achieved the following scale, with room to grow even more:
+You can start using Citus on a single node, using a distributed data model from the beginning so you are "scale out ready". When your Postgres workload needs to scale, it's easy to add worker nodes to the Citus database cluster, and/or to scale up the coordinator and worker nodes in your cluster. 
+
+Sometimes people ask "how big can Citus scale"? Here are a few examples of large-scale customers—but please keep in mind that there are many, many 2-node and 3-node Citus clusters in the wild, too.
```

Comment:
I think suggested English usage is to put the question mark inside the quotation mark.